src/components/hosts/HostListItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Badge } from "reactstrap";
+import { useTranslation } from "react-i18next";
+import { HostContext } from "components/host/HostContext";
+
+const HostStatusItem = (status) => {
+    return <Badge>status</Badge>;
+};
+
+const HostLanguageSpokenItem = (languages) => {
+    return (
+        <div>
+            {languages.map((language, index) => (
+                <Badge>{language}</Badge>
+            ))}
+        </div>
+    );
+};
+
+const HostCommentsItem = (comment) => {
+    let shorterString = comment.slice(0, 255);
+    if (comment.length > shorterString.length) {
+        shorterString = shorterString.concat(" (...)");
+    }
+    return <p>{shorterString}</p>;
+};
```

Comment:
I appreciate the idea, but this needs some better implementation. Let's not cut strings like this, as:
- this will cut things in the middle of the words
- we can think of some less cumbersome and i18n friendly solution
- from UX perspective it's better if the Volunteer has a complete description in front of their eyes